app/controllers/application_controller.rb

```diff
@@ -35,7 +35,7 @@ def default_serializer_options
   end
 
   def ensure_http_referer_is_set
-    request.env['HTTP_REFERER'] ||= '/'
+    request.env["HTTP_REFERER"] ||= "/"
```

Comment:
Was it intentional to clean this up in this PR as well? Personally I prefer "one PR == one feature" and then have cleanup PR's afterwards. Otherwise it's hard to see what change is coming from where and why.